src/gradata/_migrations/001_add_tenant_id.py

```
@@ -0,0 +1,321 @@
+        conn, "events", "schema_version"
+    ):
+        actions.append("ALTER events ADD schema_version INTEGER DEFAULT 1")
+
+    return {
+        "actions": actions,
+        "row_backfills": row_backfills,
+        "total_rows_to_backfill": sum(c for _, c in row_backfills),
+    }
+
+
+def up(conn: sqlite3.Connection, tenant_id: str) -> dict:
```

Ruff (0.15.10)

[warning] Too many branches (14 > 12)

(PLR0912)